runtime/oti/j9nonbuilder.h

```diff
@@ -1579,9 +1579,10 @@ typedef struct J9ExceptionHandler {
 	U_32 exceptionClassIndex;
 } J9ExceptionHandler;
 
-#if defined(__xlC__) || defined(J9ZOS390)  /* Covers: Z/OS, AIX, Linux PPC*/
+//JACKIE: not sure where defined(__open_xl__) condition should go
```

Comment:
C++ style comments cannot be used in a C header file (not that this one should stay anyway).